packages/3box-essentials/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@datamodels/3box-essentials",
```

Comment:
Should this not be `selfid-essentials`? 
Is `@datamodels/*` not taken? Nice.

Reply:
Yes I registered `@datamodels` after we settled on this name so we could use it like `@types` eventually.
The name here is temporary, I agree we should change it and likely also review the scope of what should get there, for example the 3ID uses the keychain and not the other definitions, while Self.ID does the opposite so we could probably split the concerns here.

Reply:
As discussed with @michaelsena, we'll setup a dedicated repo for these datamodels so this package won't be used/published as-is.